coqdoc/udoc.ml

```diff
@@ -10,6 +10,7 @@
 
 open Cdglobals
 open Printf
+open Stdlib
```

Comment:
Thanks @CohenCyril , you can just remove the pervasives qualification below if you want the code to work in older OCaml.

Reply:
ok thanks